lib/cmds/space_cmds/export.js

```diff
@@ -159,9 +157,15 @@ const exportSpace = async argv => {
     // contentful-import and contentful-export
     // expect a string for the proxy config
     // and create agents from it
-    options.proxy = proxyObjectToString(proxy)
+    if (typeof proxy !== 'string') {
```

Comment:
why not put this check in the `proxyObjectToString` function itself? :)

Reply:
I updated the proxyObjectToString to confirm that the arg is correct, but the check here seems correct to me because it distinguishes between two correct cases.